app/code/Magento/InventoryCatalog/README.md

```diff
@@ -0,0 +1,5 @@
+# InventoryCatalog
+
+**InventoryCatalog** provides implementation for bridge between **Inventory** and **Catalog**.
```

Comment:
We need to describe in more details what does "bridge" means in current context.

Because we introducing this module for better modularity and separation of concerns